src/extension.ts

```diff
@@ -15,7 +15,7 @@ const watch = (
   }
 
   const watcher = vscode.workspace.createFileSystemWatcher(
-    new vscode.RelativePattern(vscode.workspace.rootPath, "**/*.md"),
+    new vscode.RelativePattern(vscode.workspace.rootPath, "**/*{" + getFileTypesSetting().join(",") +"}"),
```

Comment:
```suggestion
    new vscode.RelativePattern(vscode.workspace.rootPath, `**/*{${getFileTypesSetting().join(",")}}`),
```

Let's use template literal here.